ClosedXML.Tests/ClosedXML.Tests.csproj

```diff
@@ -22,11 +22,11 @@
 
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
 		<PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
 		<PackageReference Include="NUnit" Version="3.13.3" />
 		<PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
-		<PackageReference Include="SonarAnalyzer.CSharp" Version="8.38.0.46746">
+		<PackageReference Include="SonarAnalyzer.CSharp" Version="8.40.0.48530">
```

Comment:
I would prefer a wildcard 8.* in order to not have to check this line every so often.
I do not know, if things could break in new versions by a changed rule-set in the analyzer